subsys/bluetooth/host/adv.c

```diff
@@ -1332,6 +1357,26 @@ int bt_le_ext_adv_delete(struct bt_le_ext_adv *adv)
 #endif /* defined(CONFIG_BT_EXT_ADV) */
 
 
+static void adv_timeout(struct k_work *work)
+{
+	int err;
+#if defined(CONFIG_BT_EXT_ADV)
+	struct k_work_delayable *dwork;
+
+	dwork = k_work_delayable_from_work(work);
+	struct bt_le_ext_adv *adv = CONTAINER_OF(work, struct bt_le_ext_adv,
+	timeout_work);
+
```

Comment:
As a general rule, we declare variables at top of scope:

```suggestion
	struct k_work_delayable *dwork;
	struct bt_le_ext_adv *adv;

	dwork = k_work_delayable_from_work(work);
	adv = CONTAINER_OF(work, struct bt_le_ext_adv, timeout_work);

```

Reply:
Oh, I missed that after adding `k_work_delayable_from_work`. Also, I initialized err to 0, to satisfy compiler warning.